packages/experimental/Switch/src/Switch.tsx

```diff
@@ -45,13 +45,15 @@ export const Switch = compose<SwitchType>({
       const { label, offText, onText, labelPosition, ...mergedProps } = mergeProps(switchInfo.props, final);
       const onOffText = switchInfo.state.toggled ? onText : offText;
       const displayOnOffText = !!offText || !!onText;
+      const isReduceMotionEnabled = AccessibilityInfo.isReduceMotionEnabled;
+      const thumbAnimation = isReduceMotionEnabled ? { animationClass: 'Ribbon_SwitchThumb' } : null;
```

Comment:
This is win32 only right? Should the Switch be stubbed on other endpoints or should this be gated on a Platform check?

Reply:
Uhh, right now my scope for this is just Win32. I haven't considered what differences the Switch will have across platforms so I think I'll just stub it on other endpoints.